src/wslview.sh

```diff
@@ -57,7 +57,7 @@ if [[ "$lname" != "" ]]; then
 		properfile_full_path="$(readlink -f "${lname}")"
 	fi
 	debug_echo "properfile_full_path: $properfile_full_path"
-	cmd="\"$(wslpath -w "${properfile_full_path:-lname}" 2>/dev/null || echo "$lname")\""
+	cmd="\"$(wslpath -w "${properfile_full_path:-$lname}" 2>/dev/null || echo "$lname")\""
```

Comment:
`wslpath -w` doesn't seem to error out when `lname` is an url (e.g. https://google.com). It produces invalid output on my system (nixos distro) where `://`translates to an non-ascii character.

Ref #243

Reply:
@crossing Interesting. I do see this behavior now. I will open an issue for it.